treeherder/perf/models.py

```diff
@@ -378,6 +382,38 @@ def autodetermine_status(self):
 
         return PerformanceAlertSummary.DOWNSTREAM
 
+    def time_to_triage(self, okr=3):
+        if self.triage_due:
+            return self.triage_due
+
+        import datetime
+
+        due = self.created
+
+        # if the alert was created in weekend, move the date to Monday
+        if due.isoweekday() == 6:
```

Comment:
It's easier to read as a reviewer if you assign weekdays variable names to the numbers beforehand e.g. `saturday = 6`